recipes-app/autoresize-console-window/files/autoresize-console-window.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#
+# Copyright (c) Siemens AG, 2021
+#
+# Authors:
+#  Chao Zeng <chao.zeng@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+
+current_settings=$(stty -g)
+stty raw -echo min 0 time 5
+printf '\0337\033[r\033[999;999H\033[6n\0338' > /dev/tty
+IFS='[;R' read -r _ rows cols _ < /dev/tty
+stty "$current_settings"
+stty cols "$cols" rows "$rows"
```

Comment:
I wonder if that parsing etc. wouldn't be simpler in python directly. Possibly, there is termios API that even delivers you cols/rows directly.